homeassistant/setup.py

```diff
@@ -374,6 +377,9 @@ def log_error(msg: str, exc_info: Exception | None = None) -> None:
             )
             return False
 
+        if not translation.async_translations_loaded(hass, integration_set):
+            await translation.async_load_integrations(hass, integration_set)
+
```

Comment:
This should likely never happen at startup since we should have already loaded them in https://github.com/home-assistant/core/blob/795d90eb1b49728e470167c4ad7c5ec6e166d000/homeassistant/bootstrap.py#L723

We should only hit this path if they install a CC and create a config flow

Reply:
It could also happen that the background task is not done yet. It would then delay component setup. I think that should be fine.

Reply:
Than we could await the task at the end of the component setup to make sure its done, and since its very unlikely it will ever happen it doesn't matter if we create some more task overhead because the system is probably already so swamped anyways it won't make a difference